ts/features/wallet/onboarding/cobadge/screens/search/ko/CoBadgeKoServiceError.tsx

```diff
@@ -30,10 +32,12 @@ const loadLocales = () => ({
  */
 const CoBadgeKoServiceError: React.FunctionComponent<Props> = props => {
   const { headerTitle, title, body } = loadLocales();
-
+  // disable hardware back
+  useHardwareBackButton(() => true);
```

Comment:
why this is needed? Can you leave also a comment about this?

Reply:
I thought on this and instead of inhibiting the back (this is a leaf screen so by design it is not possible to go back) I made him dispatch the cancel action to exit the workunit.
I've made this change in the top pr https://github.com/pagopa/io-app/pull/2774 with commit https://github.com/pagopa/io-app/pull/2774/commits/6d9e6b59b6ac285f3525775f55628ff00c09a2d8 in order to avoid conflict.